rpcs3/Emu/NP/rpcn_config.cpp

```diff
@@ -106,6 +106,255 @@ std::vector<std::pair<std::string, std::string>> cfg_rpcn::get_hosts()
 	return vec_hosts;
 }
 
+std::string cfg_rpcn::get_country() const
+{
+	return country.to_string();
+}
+
+std::vector<cfg_rpcn::country_code> cfg_rpcn::get_countries()
+{
+	std::vector<country_code> countries;
+	countries.push_back({"United States", {'u', 's'}});
```

Comment:
Do you mean outside of the function or just declaring it const there?

Reply:
Since the function is only used in one spot just replace the call to get_countries with a static constexpr definition there?

```cpp
static constexpr std::vector<country_code> countries = {
    {"United States", {'u', 's'}},
    {"Japan", {'j', 'p'}},
    ...
};
```

Or you can keep the function and just make it return a const std::vector<country_code>& and put the static constexpr definition in there and return countries;.

Reply:
This doesn't even need to be a vector, if it's const it can be just a plain C array.